ui/BoxHighlightAppIconHoverOptions.ui

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You forgot to add this file to `Makefile`

Reply:
Good catch, will update :+1: